Source/Core/Core/HW/SI/SI_DeviceGCController.h

```diff
@@ -65,10 +65,9 @@ class CSIDevice_GCController : public ISIDevice
   };
 
   // struct to compare input against
-  // Set on connection and (standard pad only) on button combo
-  SOrigin m_origin;
-
-  bool m_calibrated = false;
+  // Set in c-tor to perfect neutral values
```

Comment:
I'd avoid mentioning specific source locations, since someone will inevitably forget to remove/amend this portion of the comment if that behavior ever in changes in the future for whatever reason.

Reply:
:+1: